tsconfig.json

```diff
@@ -15,7 +15,8 @@
     "esModuleInterop": true /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */,
     "experimentalDecorators": true /* Enables experimental support for ES7 decorators. */,
     "forceConsistentCasingInFileNames": true /* Disallow inconsistently-cased references to the same file. */,
-    "resolveJsonModule": true
+    "resolveJsonModule": true,
+    "jsx": "react-jsx"
```

Comment:
Can we have a global tsconfig.json for the monorepo (at like a base file)?

Reply:
yes, this one is global and placed in the root

Reply:
sorry ... didn't look where this file is located, just saw that we have the same definition in another folder 🤦 .... that is why you don'T do reviews in the middle of the night